### ISSUE TITLE
Remove unneeded project dependencies

### ISSUE THREAD

**Issue by AndrewBogdanovTSS** [nuxt-community/typescript-template#34]
I don't see any reason for keeping these dependencies as root level:
```
"vue": "^2.5.13",
"vue-meta": "^1.4.2",
"vuex": "^3.0.1",
"vue-router": "^3.0.1",
"webpack": "^3.10.0"
```
all of those already have been installed by `nuxt` internally. 

as I've highlighted in this issue #31 these dependencies won't be needed when we'll move to `nuxt-property-decorator`
```
"vue-property-decorator": "^6.0.0",
"vue-class-component": "^6.1.2",
"nuxt-class-component": "^1.2.0",
```

These dependencies are also under a question since they don't really relate to a TypeScript template idea and might be made optional to install during template initialization process for users who really like them to be in their template. ( I personally would also like to have SASS support as an option) #32 

```
"gsap": "^1.20.3",
"tachyons": "^4.9.1",
"axios": "^0.17.1",
```

So, ideally, we should have only these dependencies installed by default:
```
"dependencies": {
    "nuxt": "^1.3.0",
    "nuxt-property-decorator": "^1.0.0",
    "vuex-class": "^0.3.0"
  },
```

<div align="right"><sub><em>This question is available on <a href="https://nuxtjs.cmty.io">Nuxt.js</a> community (<a href="https://nuxtjs.cmty.io/nuxt-community/typescript-template/issues/c28">#c28</a>)</em></sub></div>

**Comment by qm3ster** [comment]
We need to fix #29 first, and I don't understand it at all.

**Comment by hmsk** [comment]
Close with #29
